scripts/dvm

```diff
@@ -272,7 +279,7 @@ _macos_dartium_arch() {
 
 dvm_install() {
   if [[ $# < 1 ]]; then
-    echo "usage: dvm install <version>"
+    echo "usage: dvm install <version> <channel>"
```

Comment:
I should probably have always listed `VERSION` in all-caps to be consistent with common usage notation indicating required arguments; I can do that across the board as a followup, but for consistency with other optional arguments (such as the `[--path]` option in `alias`, let's do this for now:
```suggestion
    echo "usage: dvm install <version> [CHANNEL]"
```